File: CSharp/Tests/DrinksTest.cs
using Xunit;

namespace CSharp.Tests
{
    public class DrinksTest
    {
        [Theory]
        [InlineData(17, true, false)]
        [InlineData(30, true, false)]
        [InlineData(24, false, true)]
        [InlineData(18, false, true)]
        [InlineData(16, false, false)]
        [InlineData(18, true, false)]
        [InlineData(17, false, false)]
        public void ShouldServeDrinks_IntAgeBoolOnBreak_ReturnBoolean(int age, bool onBreak, bool expected)
        {
            var actual = Drinks.ShouldServeDrinks(age, onBreak);

            Assert.Equal(expected, actual);
        }
    }
}

File: js/tests/drinks.test.js
const shouldServeDrinks = require("../drinks");

const data = [
  [17, true, false],
  [30, true, false],
  [24, false, true],
  [18, false, true],
  [3, false, false]
];

describe("shouldServeDrinks", () => {
  describe("Return boolean whether the bartender should serve drinks", () => {
    test.each(data)(
      "shouldServeDrinks(%i, %s) = %s",
      (age, onBreak, expected) => {
        let actual = shouldServeDrinks(age, onBreak);

        expect(actual).toEqual(expected);
      }
    );
  });
});
